collectors/auth0/auth0_collector.js

```diff
@@ -24,6 +24,8 @@ const tsPaths = [
     { path: ['date'] }
 ];
 
+const HOSTNAME_REGEXP = /^[https]*:\/\/|\/$/g;
```

Comment:
Why repeat the t here? A bit confusing...

Reply:
I left it for readability to highlight that we'd like to remove `http/https` prefixes.
Do you think it is confusing?

Reply:
:-) I guess it depends on the reader